symengine/logic.cpp

```diff
@@ -352,6 +352,20 @@ RCP<const Boolean> logical_not(const RCP<const Boolean> &s)
     }
 }
 
+RCP<const Boolean> logical_xor(const set_boolean &s)
+{
+    int cnttrue = 0;
+    for (auto &a : s) {
+        if (eq(*a, *boolTrue)) {
+            cnttrue++;
+        }
+    }
```

Comment:
This function only checks for trues and false. `logical_xor(x, y)` would give false.